src/components/AxisCollection/CollapsedAxis.js

```diff
@@ -58,6 +61,40 @@ export default class CollapsedAxis extends Component {
     }
   }
 
+  getPath = ({ offsetx, strokeWidth, tickSizeOuter, range }) => {
+    const { yAxisPlacement, width } = this.props;
+    switch (yAxisPlacement) {
+      case AxisPlacement.LEFT:
+        return [
+          // Move to this (x,y); start drawing
+          `M ${width - offsetx} ${strokeWidth}`,
+          // Draw a horizontal line to the left
+          `h ${tickSizeOuter - strokeWidth}`,
+          // Draw a vertical line from top to bottom
+          `v ${range[0] - strokeWidth * 2}`,
+          // Finish with another horizontal line
+          `h -${tickSizeOuter - strokeWidth / 2}`,
+        ].join(' ');
+      case AxisPlacement.BOTH:
+        throw new Error(
+          'BOTH is not a valid option for CollapsedAxis -- please specify RIGHT or LEFT'
+        );
```

Comment:
What is the scenario when this will throw an error and why?

Reply:
In case we (griff developers) just blindly pass in `yAxisPlacement` without saying *which* side it's on.

For example, if the user sets the default to be `BOTH`, and then the developer just passes that into `CollapsedAxis`, rather than making a `LEFT` version and a `RIGHT` version. This will help us catch it in the storybook.